README.md

```diff
@@ -21,7 +21,8 @@ Organizations below are **officially** using Argo CD. Please send a PR with your
 
 1. [Intuit](https://www.intuit.com/)
 2. [Yieldlab](https://www.yieldlab.de/)
+3. [KompiTech GmbH](https://www.kompitech.com/)
```

Comment:
Hi, I know it's not stated, but we would like this to be in alphabetical order.

Reply:
Done